third_party/llm_engine/llm_engine.bzl

```diff
@@ -99,15 +101,15 @@ cmake(
         "HTTPS_PROXY": "{https_proxy}",
     }},
     lib_source = ":all_srcs",
-    out_lib_dir = "{lib_path}",
-   
+    out_lib_dir = "runtime/{lib_path}/intel64",
+    out_include_dir = "runtime/include",
     # linking order
-    out_static_libs = [
-            "libopenvino_continuous_batching.a",
+    out_shared_libs = [
+            "libopenvino_genai.so.2430",
```

Comment:
Does building produce `libopenvino_genai.so` ? 
Using name with `2430` will cause building issues when building with newer version and will force us to update this section.